src/auth/oauth2client.ts

```diff
@@ -49,17 +54,17 @@ export class OAuth2Client extends AuthClient {
    * @param {string=} clientId The authentication client ID.
    * @param {string=} clientSecret The authentication client secret.
    * @param {string=} redirectUri The URI to redirect to after completing the auth request.
-   * @param {Object=} opt_opts optional options for overriding the given parameters.
+   * @param {Object=} optOpts optional options for overriding the given parameters.
    * @constructor
    */
   constructor(
       clientId?: string, clientSecret?: string, redirectUri?: string,
-      opt_opts?: any) {
+      optOpts?: any) {
```

Comment:
just drop the `opt` prefix now. he function type is sufficient.

Reply:
fixed